app/models/solr_document.rb

```diff
@@ -1,22 +1,22 @@
 # frozen_string_literal: true
 
-class SolrDocument
-  include Blacklight::Solr::Document
 
-  # self.unique_key = 'id'
+module ExportXml
+  def self.extended(document)
+    document.will_export_as(:xml)
+  end
 
-  # Email uses the semantic field mappings below to generate the body of an email.
-  SolrDocument.use_extension(Blacklight::Document::Email)
+  def export_as_xml
+    self["payload_ss"]
+  end
 
-  # SMS uses the semantic field mappings below to generate the body of an SMS email.
-  SolrDocument.use_extension(Blacklight::Document::Sms)
+  alias_method :to_xml, :export_as_xml
+end
 
-  # DublinCore uses the semantic field mappings below to assemble an OAI-compliant Dublin Core document
-  # Semantic mappings of solr stored fields. Fields may be multi or
-  # single valued. See Blacklight::Document::SemanticFields#field_semantics
-  # and Blacklight::Document::SemanticFields#to_semantic_values
-  # Recommendation: Use field names from Dublin Core
-  use_extension(Blacklight::Document::DublinCore)
```

Comment:
@ebtoner I'm highlighting this removal for you because it looks like at some point we had intended to use the BL out of the box way to export solr documents to xml, but we had never configured any of the fields.  Not sure if this was an oversight or just that we decided not to use this feature at all.

Reply:
FYI here are the list of fields that can be mapped by default: https://github.com/projectblacklight/blacklight/blob/master/app/models/concerns/blacklight/document/dublin_core.rb#L18